src/cow-react/modules/wallet/web3-react/connection/trust.tsx

```diff
@@ -22,11 +22,11 @@ const BASE_PROPS = {
 
 const [trustWallet, trustWalletHooks] = initializeConnector<Connector>(
   (actions) =>
-  new InjectedWallet({
-    actions,
-    walletUrl: WALLET_LINK,
-    searchKeywords: ['isTrust', 'isTrustWallet'],
-  })
+    new InjectedWallet({
+      actions,
+      walletUrl: WALLET_LINK,
+      searchKeywords: ['isTrust', 'isTrustWallet'],
+    })
```

Comment:
what's with these changes? by now i saw them in 3 PRs

Reply:
Its the Prettier autocommit 😄 
Then I forget to pull before I do changes and then I force push :smile